apps/judicial-system/web/src/routes/Court/RestrictionCase/Confirmation/Confirmation.tsx

```diff
@@ -32,56 +32,91 @@ import {
 } from '@island.is/judicial-system-web/src/graphql/schema'
 import { useCase } from '@island.is/judicial-system-web/src/utils/hooks'
 
+import {
+  JudgeRequestRulingSignatureModal,
+  RegistrarRequestRulingSignatureModal,
+  RulingModifiedModal,
+} from '../../components'
 import { confirmation as strings } from './Confirmation.strings'
 
-type VisibleModal = 'none' | 'rulingModifiedModal' | 'signingModal'
+type VisibleModal =
+  | 'none'
+  | 'rulingModifiedModal'
+  | 'judgeRequestRulingSignatureModal'
+  | 'registrarRequestRulingSignatureModal'
+  | 'signingModal'
 
-export const Confirmation: React.FC<React.PropsWithChildren<unknown>> = () => {
+const Confirmation: React.FC<React.PropsWithChildren<unknown>> = () => {
```

Comment:
Same as in https://github.com/island-is/island.is/pull/14109/files?diff=split&w=1#diff-571b4bfba816052ce935d532394a2e538da701f21bbae75a282784d486cc3bc2R49